sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/mlflow-model-local-inference-test.ipynb

```diff
@@ -0,0 +1,281 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load the test data into a Pandas DataFrame\n",
+    "\n",
+    "Load some test images into a Pandas DataFrame in order to try some predictions with it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_image_paths = [\n",
+    "    \"./data/odFridgeObjectsMask/images/31.jpg\",\n",
+    "    \"./data/odFridgeObjectsMask/images/62.jpg\",\n",
+    "    \"./data/odFridgeObjectsMask/images/93.jpg\",\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import base64\n",
+    "\n",
+    "\n",
+    "def read_image(image_path):\n",
+    "    with open(image_path, \"rb\") as f:\n",
+    "        return f.read()\n",
+    "\n",
+    "\n",
+    "test_df = pd.DataFrame(\n",
+    "    data=[\n",
+    "        base64.encodebytes(read_image(image_path)) for image_path in test_image_paths\n",
+    "    ],\n",
+    "    columns=[\"image\"],\n",
+    ")\n",
+    "test_df.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the best model in memory\n",
+    "\n",
+    "Load the model using MLflow flavor. Check MLmodel under the downloaded folder (artifact_downloads/outputs/mlflow-model). For this particular example (and for AutoML for Images scenario), MLmodel file will describe python_function flavor. We show how to load model using pyfunc flavor. For more information on MLflow flavors, visit: https://www.mlflow.org/docs/latest/models.html#storage-format\n",
+    "\n",
+    "Loading the models locally assume that you are running the notebook in an environment compatible with the model. The list of dependencies that is expected by the model is specified in the MLFlow model produced by AutoML (in the 'conda.yaml' file within the mlflow-model folder)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlflow.pyfunc"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Way #1: Get the MLFlow model from the downloaded MLFlow model files\n",
+    "\n",
+    "pyfunc_model = mlflow.pyfunc.load_model(\"./artifact_downloads/outputs/mlflow-model\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make the predictions\n",
+    "\n",
+    "result = pyfunc_model.predict(test_df).to_json(orient=\"records\")\n",
+    "result"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize detections\n",
+    "We have scored test images. Now, we'll visualize the bounding boxes for first image."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Please make sure that all the visualizations are working as expected. Asking because any errors in this code may not be caught in GitHub actions, so it might be better to double check